src/Operators/spectralelement.jl

```diff
@@ -83,12 +83,27 @@ Base.Broadcast.broadcasted(
 Base.eltype(sbc::SpectralBroadcasted) =
     operator_return_eltype(sbc.op, tuplemap(eltype, sbc.args)...)
 
+@inline instantiate_args(args::Tuple) =
+    (Base.Broadcast.instantiate(args[1]), instantiate_args(Base.tail(args))...)
+@inline instantiate_args(args::Tuple{Any}) =
+    (Base.Broadcast.instantiate(args[1]),)
+@inline instantiate_args(::Tuple{}) = ()
+
+if VERSION >= v"1.7.0"
```

Comment:
I think it would be good to check if this is really needed.

Reply:
Does it help with a particular test case?

Reply:
I'm fairly certain that the main gains were from defining `Base.copyto!(::FieldVector,::Base.Broadcast.Broadcasted)`. I'll remove this.

Reply:
We require 1.7 as of #473.

Reply:
Ah, yeah, lol